### PR TITLE
Install optional credentials requirements

### PR DESCRIPTION
**Pull request by rlucioni**
@maxrothman @zubair-arbi

### CODE REVIEW

File: playbooks/roles/credentials/tasks/main.yml [hunk 1]
```diff
@@ -30,25 +30,13 @@
...
-    - install
-    - install:app-requirements
-
 - name: install application requirements
-  command: make prod-requirements
-  args:
-    chdir: "{{ credentials_code_dir }}"
+  pip:
+    requirements: "{{ credentials_requirements_base }}/{{ item }}"
+    virtualenv: "{{ credentials_venv_dir }}"
```
Comment (maxrothman) [inline playbooks/roles/credentials/tasks/main.yml:37]: I think this requires the virtualenv already exist, which means you shouldn't remove the above task.

Comment (rlucioni) [inline playbooks/roles/credentials/tasks/main.yml:37]: Ansible should initialize the virtualenv if necessary. See http://docs.ansible.com/ansible/pip_module.html and how we do this for [programs](https://github.com/edx/configuration/blob/master/playbooks/roles/programs/tasks/main.yml#L30-L36).

**Comment by zubair-arbi** [comment]
Changes look good :+1:

File: playbooks/roles/credentials/tasks/main.yml (hunk 1 above)
Comment (rlucioni) [inline playbooks/roles/credentials/tasks/main.yml:37]: We verified this offline. The virtualenv is indeed created if it's missing.

**Comment by maxrothman** [comment]
:+1:

**Pull request merged**